### PR TITLE
Syntax Lookup: ignore() function

### PR DESCRIPTION
**Pull request by kevanstannard**
Hi team, suggestion for #162 and #448

### CODE REVIEW

File: pages/docs/manual/latest/function.mdx [hunk 1]
```diff
@@ -409,33 +409,36 @@ add(1, 2);
 
 </CodeTab>
 
-If you need to call a curried function with a single unit `()` argument, you can use the `ignore()` function:
```
Comment (kevanstannard) [inline pages/docs/manual/latest/function.mdx:412]: 🗒️ Looks like this is no longer required for v10?

File: src/SyntaxLookup.res [hunk 2]
```diff
@@ -31,13 +31,21 @@ let requireSyntaxFile: string => MdxComp.t = %raw(`
 `)
 
 module Category = {
-  type t = Decorators | Operators | LanguageConstructs | ExtensionPoints | SpecialValues | Other
+  type t =
+    | Decorators
+    | Operators
+    | LanguageConstructs
+    | BuiltInFunctions
```
Comment (kevanstannard) [inline src/SyntaxLookup.res:38]: 🗒️ New category

File: pages/docs/manual/latest/function.mdx (hunk 1 above)
Comment (cristianoc) [reply pages/docs/manual/latest/function.mdx:412]: Indeed, plus it should have said "uncurried".

**Review by cristianoc — APPROVED** [review]
This looks great, thanks for that.
Just a question: should this also show up in the overall search?

In other words, it's generally not clear to me when one should reach for the overall search (the one powered by algolia), and when one should reach out for syntax lookup.

**Comment by kevanstannard** [comment]
It's a good question. It was originally inspired by a JS syntax lookup.

Here's how I think about it at the moment:

[1] Syntax Lookup on its own provides a quick reference / cheat sheet of syntax.
[2] Syntax Lookup and Algolia are both kinds of "indexes" to help you locate primary documentation pages.

Since the Syntax Lookup only contains summarised information that is already on the primary documentation pages, perhaps it's not necessary to include in Algolia?

@ryyppy what do you think?

**Comment by ryyppy** [comment]
LGTM. Regarding Algolia: I think the widget pages should not show up in the search results for the same reasons you just mentioned.

**Pull request merged**